cmd/root/sweet.go

```diff
@@ -275,12 +183,12 @@ func runeToEventExpected(r rune) string {
 
 // Creates a new event. Should be used when recording a keystroke
 // to the model.
-func NewEvent(typed string, expected string, i int) event {
-	return event{
-		ts:       time.Now(),
-		typed:    typed,
-		expected: expected,
-		i:        i,
+func NewEvent(typed string, expected string, i int) util.Event {
```

Comment:
This function belongs closer to the Event struct in the `util` file instead of here